vms/rpcchainvm/runtime/subprocess/linux_stopper.go

```diff
@@ -30,14 +30,19 @@ func NewCmd(path string, args ...string) *exec.Cmd {
 func stop(ctx context.Context, log logging.Logger, cmd *exec.Cmd) {
 	waitChan := make(chan error)
 	go func() {
-		// attempt graceful shutdown
+		defer close(waitChan)
+		// check if process is already stopped by sending signal 0.
+		if err := cmd.Process.Signal(syscall.Signal(0)); err != nil {
+			waitChan <- nil
```

Comment:
This seems like it should be reporting an error - not reporting that shutdown succeeded.

Reply:
yeah prob not worth the additional check. otherwise would need to send os.ErrProcessDone which is probably what would happen anyways.